src/bia_bob/_machinery.py

```diff
@@ -93,3 +93,7 @@ def init_assistant(model="gpt-3.5-turbo", temperature=0):
           "below two lines into the next cell and execute it."
           "\n\n%%bob"
           "\nplease generate a noisy grayscale image containing 10 blurry blobs with a diameter of 20 pixels each.")
+
+def wtf():
```

Comment:
@haesleinhuepf that looks like it can be removed? 🤣 
...or is it actually used?

Reply:
lol

Reply:
I will replace this PR in a minute with another magic: `%%fix` instead of `bob.fix()`...